src/components/ItemsNotFoundComponent.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+import { withRouter } from 'react-router-dom';
+import { Image } from 'semantic-ui-react';
+import PropTypes from 'prop-types';
+
+import '../_css/ItemsNotFoundComponent.css';
+
+export class ItemsNotFoundComponent extends Component {
+  state = {
+    message: this.props.message
+  }
+  render() {
+    return (
+      <div className="not-found">
+        <div className="middle">
```

Comment:
Please use BEM methodology for classes. The bench projects have moved to the technology department, and the engineering handbook recommends the use of `BEM methodology` for classes.

You can read more in:
https://en.bem.info/methodology/quick-start/
http://getbem.com/introduction/